src/index.html

```diff
@@ -19,9 +22,413 @@
       rel="stylesheet"
       href="styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700;900&display=swap"
+      rel="stylesheet"
+    />
```

Comment:
There are duplicate `<link>` elements for preconnecting to Google Fonts and loading the Roboto font. Consider removing the redundant links to optimize loading performance and reduce unnecessary HTTP requests. You can consolidate the font loading to a single set of `<link>` elements.